### PR TITLE
zeek: add page

### LINKED ISSUE #1627: page request: zeek
http://bro.org

### PR DESCRIPTION
**Pull request by marchersimon**
Closes #1627

### CODE REVIEW

File: pages/common/zeek.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+- Analyse live traffic from a specific interface:
+
+`sudo zeek --iface {{interface}}`
+
+- Analyse live traffic from a specific interface and load custom scripts:
+
+`sudo zeek --iface {{interface}} {{script1}} {{script2}}`
+
+- Analyse live traffic from a specific interface loading no scripts:
```
Comment (bl-ue) [inline pages/common/zeek.md:15]:
```suggestion
- Analyse live traffic from a specific interface, without loading any scripts:
```

File: pages/common/zeek.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+`sudo zeek --bare-mode --iface {{interface}}`
+
+- Analyse live traffic from a specific interface applying a `tcpdump` filter:
```
Comment (bl-ue) [inline pages/common/zeek.md:19]:
```suggestion
- Analyse live traffic from a specific interface, applying a `tcpdump` filter:
```

**Review by bl-ue — COMMENTED** [review]
Nice! I love closing these old issues. Left a few suggestions.

File: pages/common/zeek.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# zeek
+
+> Passive network traffic analyser.
+> Any output and log files will be saved to the current working directory.
+> More information: <https://docs.zeek.org/en/lts/quickstart.html#zeek-as-a-command-line-utility>.
+
+- Analyse live traffic from a specific interface:
```
Comment (bl-ue) [inline pages/common/zeek.md:7]:
```suggestion
- Analyse live traffic from a network interface:
```

File: pages/common/zeek.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+`sudo zeek --iface {{interface}}`
+
+- Analyse live traffic from a specific interface and load custom scripts:
+
+`sudo zeek --iface {{interface}} {{script1}} {{script2}}`
+
+- Analyse live traffic from a specific interface, without loading any scripts:
```
Comment (bl-ue) [inline pages/common/zeek.md:15]:
```suggestion
- Analyse live traffic from a network interface, without loading any scripts:
```

File: pages/common/zeek.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+`sudo zeek --bare-mode --iface {{interface}}`
+
+- Analyse live traffic from a specific interface, applying a `tcpdump` filter:
```
Comment (bl-ue) [inline pages/common/zeek.md:19]:
```suggestion
- Analyse live traffic from a network interface, applying a `tcpdump` filter:
```

File: pages/common/zeek.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+`sudo zeek --filter {{path/to/filter}} --iface {{interface}}`
+
+- Analyse live traffic from a specific interface using a watchdog timer:
```
Comment (bl-ue) [inline pages/common/zeek.md:23]:
```suggestion
- Analyse live traffic from a network interface using a watchdog timer:
```

**Review by bl-ue — COMMENTED** [review]
`s/specific interface/network interface/`?

File: pages/common/zeek.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+
+> Passive network traffic analyser.
+> Any output and log files will be saved to the current working directory.
+> More information: <https://docs.zeek.org/en/lts/quickstart.html#zeek-as-a-command-line-utility>.
+
+- Analyse live traffic from a specific interface:
+
+`sudo zeek --iface {{interface}}`
+
+- Analyse live traffic from a specific interface and load custom scripts:
```
Comment (bl-ue) [inline pages/common/zeek.md:11]: Sorry missed this :)
```suggestion
- Analyse live traffic from a network interface and load custom scripts:
```

**Review by bl-ue — APPROVED** [review]

File: pages/common/zeek.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+
+`sudo zeek --filter {{path/to/filter}} --iface {{interface}}`
+
+- Analyse live traffic from a network interface using a watchdog timer:
+
+`sudo zeek --watchdog --iface {{interface}}`
+
+- Analyse traffic from a `pcap` file:
+
+`sudo zeek --readfile {{path/to/file.trace}}`
```
Comment (navarroaxel) [inline pages/common/zeek.md:29]: this require sudo?

Comment (marchersimon) [reply pages/common/zeek.md:29]: Hm, that's weird. Yesterday it didn't work without sudo, today it does. I'll remove it then.

**Review by owenvoke — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**